migrations/Version20200.php

```
@@ -75,12 +75,12 @@ public function __construct(Connection $connection, LoggerInterface $logger)
 
     public function isTransactional(): bool
     {
-        return ! $this->platform instanceof MySQLPlatform && ! $this->platform instanceof OraclePlatform;
+        return ! $this->platform instanceof AbstractMySQLPlatform && ! $this->platform instanceof OraclePlatform;
     }
 
     public function preUp(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)